cmd/feed-ingress/main.go

```diff
@@ -62,6 +71,19 @@ func configureLogging() {
 	}
 }
 
+func configureHealthPort() {
+	http.HandleFunc("/health", checkHealth)
```

Comment:
It doesn't look possible in golang, see https://github.com/golang/go/issues/4674.

We can do it using a third party library like https://github.com/facebookgo/httpdown, but it will add a lot more code. Is it necessary for the health check?